{{cookiecutter.repo_name}}/.github/workflows/main.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.6
```

Comment:
I suggest reverting this and subsequent lines, for reasons I'll explain in the review summary.

Reply:
Done in https://github.com/getpelican/cookiecutter-pelican-plugin/pull/12/commits/960948d4437760cfcaa91bdd700d0f3fe672a543